CHANGELOG.md

```diff
@@ -1,12 +1,28 @@
 # CHANGELOG
 All notable changes to this project will be documented in this file. To know better on how to write and maintain a changelog, refer to [this link](https://keepachangelog.com/en/1.0.0/).
 
+## [1.2.0]
+
+### Added
+
+- Support for Signup+.
+
+### Changed
+
+- Android SDK support target lowered to API level 21
```

Comment:
```suggestion
- Android SDK support target lowered to API level 21. Please check the [readme](README.md) for an extra, mandatory `build.gradle` setup to avoid crashes.
```

Reply:
I was the one to remove this line as I do t think this is part of a CHANGELOG. It's more of a documentation/Readme thing, no?

Reply:
This is true, but I want to make sure that whoever integrated the SDK previously will not miss this information. Not doing it will not cause the SDK to crash immediately, but only in some cases. So lack of this setup is not immediately obvious.